### ISSUE TITLE
No audio

### ISSUE THREAD

**Issue by slumbergod** [keredson/gnomecast#136]
Almost everything plays as it should but sometimes I come across a title where there is no audio. Today's viewing is one such example.

From VLC report:
Type: Audio
Codec: A52 Audio (aka AC3) (a52 )
Channels: 3F2M/LFE
Sample rate: 48000 Hz
Bits per sample: 32

Is there a way to force the audio to be encoded to mp3 so it plays without problems? Usually, I use handbrake and re-encode video and audio but that is a hassle.

**Comment by keredson** [comment]
different chromecasts support just x264 or x264 and x265, but i have no known way to query, so i've just been blacklisting devices that can't play x265.  what device do you have?  does it support 4k?

**Comment by slumbergod** [comment]
No, we don't have the 4K version. Our chromecast is a couple of years old. Maybe a solution is to add an option to force re-encoding of all audio and video and make it an option the user can set. These days when I have to manually re-encode a video to work I just choose old formats (like mp3 for audio and one of the web settings in handbrake). Then gnomecast has no problems at all.
